### PR TITLE
Add blog post for installation improvements since v2.0.

### PR DESCRIPTION
**Pull request by GeigerJ2**
Making this PR here to get feedback on the blog post and iterate on it, to have it ready once we start publishing the announcements on LinkedIn.

[Direct link](https://aiida-website--69.org.readthedocs.build/news/posts/2024-09-20-simpler-installation.html) to the rendered post on RTD.

### CODE REVIEW

**Review by agoscinski — COMMENTED** [review]
LGTM and renders good. Only concern are the gifs size. The docs in total were 21MB and now are 54MB. I think using them like this frequently is not really an option. Maybe you can use git lfs for this? https://docs.gitlab.com/ee/topics/git/lfs/#add-a-file-type-to-git-lfs, but I am not sure if this repo supports it.

**Comment by GeigerJ2** [comment]
Thanks for the review, @agoscinski. Yeah, the file size of the GIFs is a good point! We could also host those in another repo, or on Google Drive. Just need a publicly accessible URL. Played around a bit with frame rates and resolutions, and I wouldn't go lower, otherwise they don't render nicely.

**Comment by agoscinski** [comment]
Probably google drive is a better way to go then, I think LFS is limited to 1 GB.

**Comment by agoscinski** [comment]
Maybe you could try? https://imgbb.com

**Comment by GeigerJ2** [comment]
As discussed with @giovannipizzi, we think the GIF size is OK for now. If they are only in the <20mb range, and they won't be changed continuously, they won't bloat the git history. If the repo eventually grows too large, we can still move them to some external hosting service. I'd only actually rename the directory from `_static` to `_gifs`.

**Comment by giovannipizzi** [comment]
Thanks! On the content, looks OK, just a few minor changes

- There is a `:rocket:` that does not render (no emoji, only text)
- `tech-savy` -> `tech-savvy`
- in footnote [1], I would change "and should not be used in production" -> "and should not be used in production, at least for now (we are working in the next releases trying to reduce as much as possible limitations)", or something like this

For the GIFs, if you have time, trying LFS (I never did myself) is indeed interesting, assuming though that it works also with rendered pages with sphinx etc. Otherwise as I said it's not a big deal, even if I think we can do much better.
At the moment the GIF is 2160 × 1215 pixels, but even when I maximize the screen in the page, it only goes up to 720x405px; even putting a factor of 2 for retina displays, one could probably already save a factor of 3-4x by recording only a screen with size 1440x...

Then, I would also reduce the frame rate, now it's 30fps (670 frames for about 22 seconds of video), I think we can go 10fps or lower (to be tested), since anyway there are not major animations.

Maybe this already brings every video to <1MB and then I wouldn't worry too much.
But finally, one might want to play with compression software (I tried quickly both with GraphicsMagick and with an online tool, but quality went down horribly, I'm sure there are better software parameters or tools), but e.g. there should be a way to only store the diff between every frame and then next - considering that most pixels are the same, maybe one can also save a lot (unless this is already done in the current version, I'm not sure)

**Comment by GeigerJ2** [comment]
Hi @giovannipizzi, thanks for the review!

> * There is a `:rocket:` that does not render (no emoji, only text)
> * `tech-savy` -> `tech-savvy`
> * in footnote [1], I would change "and should not be used in production" -> "and should not be used in production, at least for now (we are working in the next releases trying to reduce as much as possible limitations)", or something like this

Done!

> For the GIFs, if you have time, trying LFS (I never did myself) is indeed interesting, assuming though that it works also with rendered pages with sphinx etc. Otherwise as I said it's not a big deal, even if I think we can do much better. At the moment the GIF is 2160 × 1215 pixels, but even when I maximize the screen in the page, it only goes up to 720x405px; even putting a factor of 2 for retina displays, one could probably already save a factor of 3-4x by recording only a screen with size 1440x...

Hm, that's weird. How do you obtain the pixel size? When I inspect, it shows me the original resolution I set, though, it might just be mirroring the properties of the original file... I can also definitely tell a difference in image quality on RTD between 2160p and 1080p, though, the latter still looks good enough, I think.
 
> Then, I would also reduce the frame rate, now it's 30fps (670 frames for about 22 seconds of video), I think we can go 10fps or lower (to be tested), since anyway there are not major animations.

Hadn't really tried this before, because I thought the videos would be shown slower. But this actually works! Thanks!

> Maybe this already brings every video to <1MB and then I wouldn't worry too much. But finally, one might want to play with compression software (I tried quickly both with GraphicsMagick and with an online tool, but quality went down horribly, I'm sure there are better software parameters or tools), but e.g. there should be a way to only store the diff between every frame and then next - considering that most pixels are the same, maybe one can also save a lot (unless this is already done in the current version, I'm not sure)

I don't think GIF is really made for the diff compression. That's usually how other formats like mp4 reduce the file size. I thought it wasn't even possible for GIF, but apparently `ffmpeg` has an option. I tried it, and the file size was a bit larger than without. So I think our best bet for now is to go with 10fps and 1080p, with which the file sizes are between 400K and 1.2M, and then we don't have to worry. The final result still looks nice. I'd still upload the original `mkv`s and the GIFs to the AiiDA Google Drive, and I'll compile all the information in the wiki page here, so for future blog posts, we don't have to spend so much time on this topic anymore 😅

**Comment by GeigerJ2** [comment]
Alright, I added a page about the GIF creation [to the wiki of this repo](https://github.com/aiidateam/aiida-website/wiki/Generating-GIFS-for-blog-posts), as this was a bit of a time sink to figure everything out.

Regarding the post here, I think this is ready to be merged and publicized this Friday (to be discussed in the team meeting before).

File: docs/news/posts/2024-09-20-simpler-installation.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+blogpost: true
+category: Releases
```
Comment (agoscinski) [inline docs/news/posts/2024-09-20-simpler-installation.md:3]: Isn't it a blog post?
```suggestion
category: Blog
```

**Review by agoscinski — APPROVED** [review]

**Pull request merged**